pkgs/by-name/cu/cursewords/package.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, python3Packages
+, fetchFromGitHub
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "cursewords";
+  version = "1.1";
+
+  src = fetchFromGitHub {
+    owner = "thisisparker";
+    repo = pname;
```

Comment:
this is considered anti-pattern https://github.com/NixOS/nixpkgs/issues/277994

Reply:
Oops, fixed.